x-pack/plugins/security_solution/public/timelines/components/timeline/search_or_filter/search_or_filter.tsx

```diff
@@ -136,6 +149,39 @@ export const SearchOrFilter = React.memo<Props>(
                 updateReduxTime={updateReduxTime}
               />
             </EuiFlexItem>
+            <EuiFlexItem grow={false}>
+              <EuiFlexGroup
+                direction="row"
+                gutterSize="none"
+                alignItems="center"
+                justifyContent="center"
+              >
+                <EuiToolTip content={dataProviderIconTooltipContent}>
+                  <EuiButtonIcon
```

Comment:
I wonder if it would be helpful to include a count next to this as well for the number of data providers that are currently present? so `2 [Timeline Icon here]`

Reply:
Actually I tried, but there were 2 issues:

1. data providers can be nested. Do we want to count them as saperate data providers.
2. There were styling issues as number badges were very small when compared to the button and hence I did not do it. Let me spend some more time on that today.

Reply:
What do you think of this: 
![image](https://github.com/elastic/kibana/assets/7485038/a77ecb02-f069-4ce7-951c-e72a5f388036)

Reply:
or we can simple show a dot as well instead of a number.

Reply:
A dot might work instead of a number. Like an "active" dot or something